Scripts/Web_Scrappers/Amazon_products_scraper/requirements.txt

```diff
@@ -0,0 +1,2 @@
+requests

```

Comment:
Add specific version for used module and packages

use freeze list in your virtual environment to create requirements.txt file
```
pip freeze list>requirements.txt
```

Reply:
Updated it. Thanks for the suggestion @chavarera